### PR TITLE
(docs): tags and version update, Nx docs, IP address whitelisting

### PR DESCRIPTION
**Pull request by zmzlois**
### What's added in this PR?

- Tags and version update
- Whitelisting IP address
- Update Nx docs

### What's the issues or discussion related to this PR (optional) ?

[_Provide some background information related to this PR, including issues or task. Prior to this PR what's the behavior that wasn't expected._]

[_If there wasn't discussion related to this PR, you can include the reasoning behind this PR of why you did it._]

### Who do you wish to review this PR other than required reviewers?

d
### (Required) Pre-PR/Merge checklist

- [ ] I have added/updated our feature to sync with this PR
- [ ] I have added an explanation of my changes
- [ ] I have/will run tests, or ask for help to add test

### CODE REVIEW

File: docs/how-to/allow-ip-addresses.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Allow IP Addresses 
+
+:::info 
+This guide is recommended for users bring-your-own-cloud(BYOC) and using our managed cloud. Zephyr Cloud leverages many other IP ranges for various product and services but some of these address will not make connections to your origin. 
+::: 
+
+Zephyr has several IP addresses which are used by all proxied hostnames. These IP addresses construct the skeleton of our global edge network to help distribute traffic amongs various servers.
```
Comment (swalker326) [inline docs/how-to/allow-ip-addresses.mdx:7]: among various servers

File: docs/how-to/allow-ip-addresses.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Allow IP Addresses 
+
+:::info 
+This guide is recommended for users bring-your-own-cloud(BYOC) and using our managed cloud. Zephyr Cloud leverages many other IP ranges for various product and services but some of these address will not make connections to your origin.
```
Comment (swalker326) [inline docs/how-to/allow-ip-addresses.mdx:4]: `bringing-your-own-cloud` - idk if this needs reworded in general. 

This guide is recommended for users who want to "bring your own cloud" (BYOC) or using our managed cloud.

File: docs/how-to/allow-ip-addresses.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Allow IP Addresses 
+
+:::info 
+This guide is recommended for users bring-your-own-cloud(BYOC) and using our managed cloud. Zephyr Cloud leverages many other IP ranges for various product and services but some of these address will not make connections to your origin. 
+::: 
+
+Zephyr has several IP addresses which are used by all proxied hostnames. These IP addresses construct the skeleton of our global edge network to help distribute traffic amongs various servers. 
+
+
+If you have private network setup, implemented IP filtering, or allowing certain IP addresses in your organization, all traffic between your origin and Zephyr Cloud's server will pass through your proxy network. Without whitelisting below IP addresses to your allow list, devices at your origin will stop receiving traffic from our servers. Blocking these IPs will resulting visitor traffic termination or deployment failure. Whitelisting below IP addresses enable connections to our backend server and providers to enable flawless deployment flow.
```
Comment (swalker326) [inline docs/how-to/allow-ip-addresses.mdx:10]: or are allowing

Comment (swalker326) [inline docs/how-to/allow-ip-addresses.mdx:10]: Blocking these IPs will result in

File: docs/how-to/versioning-tags.mdx [hunk 4]
```diff
@@ -1,8 +1,8 @@
-# Versioning and Tags
+# Versioning, Tags and Environments
 
 Zephyr utilizes familiar versioning concepts by adopting [SemVer](https://semver.org/) (Semantic Versioning) as its foundational versioning methodology, ensuring a systematic and meaningful progression of project versions.
 
-Applications deployed through Zephyr could have unlimited number of versions. After you configure Zephyr's npm plugin in your build tool, or bundlers, _**each successful deployment would generate a new version**_. 
+Applications deployed through Zephyr could have unlimited number of versions. After you configure Zephyr's npm plugin in your build tool, or bundlers, _**each successful deployment would generate a new version**_. Check [here to see how tags are created](#how-tags-are-created) and [how versions, tags and environments work together](#).
```
Comment (swalker326) [inline docs/how-to/versioning-tags.mdx:5]: can have an unlimited number

Comment (swalker326) [inline docs/how-to/versioning-tags.mdx:5]: **each successful deployment will generate a new version**

File: docs/recipes/vite-rspack-webpack-mf.mdx [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+```
+
+For applications built with Webpack and Rspack: 
+
+```sh title="Terminal"
+npm i zephyr-webpack-plugin@latest
+```
+
+## Example configuration
+Four example build configuration for Vite, Rspack and Webpack.
```
Comment (swalker326) [inline docs/recipes/vite-rspack-webpack-mf.mdx:36]: For example build

File: docs/how-to/allow-ip-addresses.mdx (hunk 2 above)
Comment (zackarychapple) [reply docs/how-to/allow-ip-addresses.mdx:4]: I agree with @swalker326

File: docs/recipes/existing-app.mdx [hunk 6]
```diff
@@ -31,7 +31,7 @@ If you already have an application and wish to enhance it with Zephyr Cloud, we
 ### 1. Add Zephyr as a dependency
 
 Install the package via NPM:
-<Tabs values={[{label:"Rspack"}, {label: 'Webpack'}, {label: 'Vite'}]}>
+<Tabs values={[{label:"Rspack"}, {label: 'Webpack'}]}>
```
Comment (zackarychapple) [inline docs/recipes/existing-app.mdx:34]: brand guidelines for webpack are that it should always be lower case `webpack`

File: rspress.config.ts [hunk 7]
```diff
@@ -132,6 +136,10 @@ const sidebar: UserConfig['themeConfig']['sidebar'] = {
           text: 'React + Vite',
           link: '/recipes/react-vite',
         },
+        {
+          text: 'Vite + Rspack + Webpack + MF',
```
Comment (zackarychapple) [inline rspress.config.ts:140]: same here on `webpack`

File: docs/how-to/allow-ip-addresses.mdx (hunk 2 above)
Comment (zmzlois) [reply docs/how-to/allow-ip-addresses.mdx]: https://github.com/ZephyrCloudIO/zephyr-documentation/pull/27/commits/1306d2569bd71baff915d22883f40f719909844f

File: docs/how-to/allow-ip-addresses.mdx (hunk 1 above)
Comment (zmzlois) [reply docs/how-to/allow-ip-addresses.mdx]: https://github.com/ZephyrCloudIO/zephyr-documentation/pull/27/commits/336fa70a5d86e98a4b3f89def2197d4146ec4bbe

File: docs/how-to/allow-ip-addresses.mdx (hunk 3 above)
Comment (zmzlois) [reply docs/how-to/allow-ip-addresses.mdx]: https://github.com/ZephyrCloudIO/zephyr-documentation/pull/27/commits/3c49b2e973ba95adde984411713e5fd32c56010a

Comment (zmzlois) [reply docs/how-to/allow-ip-addresses.mdx]: https://github.com/ZephyrCloudIO/zephyr-documentation/pull/27/commits/8fbd027be7a743c7ace3812957030d31b04d8ef5

File: docs/how-to/versioning-tags.mdx (hunk 4 above)
Comment (zmzlois) [reply docs/how-to/versioning-tags.mdx]: https://github.com/ZephyrCloudIO/zephyr-documentation/pull/27/commits/0b59ec33a8230d9efa2dc12717228ef8b186a6fb

File: docs/recipes/vite-rspack-webpack-mf.mdx (hunk 5 above)
Comment (zmzlois) [reply docs/recipes/vite-rspack-webpack-mf.mdx]: https://github.com/ZephyrCloudIO/zephyr-documentation/pull/27/commits/109c740762d8a4f833b0aa9f1a41810e592013fe

File: rspress.config.ts (hunk 7 above)
Comment (zmzlois) [reply rspress.config.ts]: https://github.com/ZephyrCloudIO/zephyr-documentation/pull/27/commits/37191a7d5a82b2a2987d1331cf2f64e100b4aa42

**Review by StashBank — APPROVED** [review]

**Pull request merged**